README.md

```diff
@@ -9,3 +9,4 @@
 ## Completed
 
 - [x] ARYAN: I will prepare the dataset -- downloading + loading them up in notebook
+- [x] MKO: I will add some of the suggested topics (datasets) -- downloading + loading them up in notebook
```

Comment:
@katushk-a @madanela 
Do we really need all this data to work on?
I thought we will pick one or two extra topics to check correlation.